### ISSUE TITLE
Sublime as default text editor

### ISSUE THREAD

**Issue by oberrauch** [sb2nov/mac-setup#220]
### I'm Submitting a ...

```
[ ] Bug report
[x] Tool/language/etc documentation request
```

### Tool/Language/etc

I think it is handy to set Sublime Text as default text editor, in order to open any plain text file in Sublime via the command `open -t foo.txt`.
This is pretty easily done, by changing the Apples default launch service `defaults write com.apple.LaunchServices LSHandlers -array-add '{LSHandlerContentType=public.plain-text;LSHandlerRoleAll=com.sublimetext.3;}'` (for Sublime Text 3).
So maybe you want to include this into your guide...

I did not come up with this, all credits to [John Staff at Coderwall](https://coderwall.com/p/jefqyg/setting-sublime-text-as-the-default-editor-osx). 

### Other Information

P.S.: Great job with this guide, and many thanks!

**Comment by Kyslik** [comment]
Hmm, I just use `subl file.txt` or `code file.txt`.

**Comment by oberrauch** [comment]
Yeah,sure. I just got used to it and so I though it would be an useful addition. People than can than decide for them selfs, but at least it is noted and they don't have to search for it somewhere else.

**Comment by simeg** [comment]
I don't think we should assume people use Sublime. Me for example have never used it. I think it's better to mention how to open sublime using via the terminal, like @Kyslik shows, and if you want to have `open` as `subl` you can just have an alias for it, no need to change anything in the OS.

`alias open="subl"`
